File: app/src/main/java/br/com/jonathanzanella/myexpenses/sync/ServerData.java
package br.com.jonathanzanella.myexpenses.sync;

import android.annotation.SuppressLint;
import android.content.Context;
import android.content.SharedPreferences;
import android.preference.PreferenceManager;

public class ServerData {
	private static final String SERVER_URL = "ServerUrl";
	private static final String SERVER_TOKEN = "ServerToken";

	private SharedPreferences sharedPreferences;

	public ServerData(Context context) {
		sharedPreferences = PreferenceManager.getDefaultSharedPreferences(context);
	}

	@SuppressLint("ApplySharedPref")
	public void updateInfo(String serverUrl, String serverToken) {
		SharedPreferences.Editor editor = sharedPreferences.edit();
		editor.putString(SERVER_URL, serverUrl);
		editor.putString(SERVER_TOKEN, serverToken);
		editor.commit();
	}

	public String getServerUrl() {
		return sharedPreferences.getString(SERVER_URL, "");
	}

	public String getServerToken() {
		return sharedPreferences.getString(SERVER_TOKEN, "");
	}
}

File: app/src/main/kotlin/br/com/jonathanzanella/myexpenses/sync/ServerData.kt
package br.com.jonathanzanella.myexpenses.sync

import android.annotation.SuppressLint
import android.content.Context
import android.content.SharedPreferences
import android.preference.PreferenceManager

class ServerData(context: Context) {

    private val sharedPreferences: SharedPreferences = PreferenceManager.getDefaultSharedPreferences(context)

    @SuppressLint("ApplySharedPref")
    internal fun updateInfo(serverUrl: String, serverToken: String) {
        val editor = sharedPreferences.edit()
        editor.putString(SERVER_URL, serverUrl)
        editor.putString(SERVER_TOKEN, serverToken)
        editor.commit()
    }

    val serverUrl: String?
        get() = sharedPreferences.getString(SERVER_URL, null)

    val serverToken: String?
        get() = sharedPreferences.getString(SERVER_TOKEN, null)

    companion object {
        private val SERVER_URL = "ServerUrl"
        private val SERVER_TOKEN = "ServerToken"
    }
}
